File: C/BOJ_2563_색종이.c
#include <stdio.h>

int main()
{
    int x, y, num, count = 0, table[100][100] = { 0, };
    scanf("%d", &num);
    for (int i = 0; i < num; i++){
        scanf("%d %d", &x, &y);
        for (int j = x - 1; j < x + 9; j++){
            for (int k = y - 1; k < y + 9; k++){
                table[j][k] = 1;
            }
        }
    }

    for (int i = 0; i < 100; i++){
        for (int j = 0; j < 100; j++){
            if (table[i][j] == 1) count++;
        }
    }

    printf("%d\n", count);
    return 0;
}

File: Python/BOJ_2563_색종이.py
import sys

table = [[0 for _ in range(100)] for _ in range(100)]
paper = int(sys.stdin.readline())
for n in range(paper):
    x, y = map(int, sys.stdin.readline().split())
    for a in range(x - 1, x + 9):
        for b in range(y - 1, y + 9):
            if table[a][b] == 0:
                table[a][b] = 1

count = 0
for i in range(100):
    for j in range(100):
        if table[i][j] == 1:
            count += 1

print(count)